styleguide/_sass/_rsweb-font-style.scss

```diff
@@ -1,10 +1,10 @@
 @font-face {
     font-family: 'rsweb';
-    src:    url('../fonts/rsweb.eot?ezv9m0');
-    src:    url('../fonts/rsweb.eot?ezv9m0#iefix') format('embedded-opentype'),
-        url('../fonts/rsweb.ttf?ezv9m0') format('truetype'),
-        url('../fonts/rsweb.woff?ezv9m0') format('woff'),
-        url('../fonts/rsweb.svg?ezv9m0#rsweb') format('svg');
+    src:    url('https://0d0fbe434114a2d40b45-de9ed8c45372075d79906316dada8cf3.ssl.cf1.rackcdn.com/icon/rsweb.eot?ezv9m0');
```

Comment:
No clue on the querystring.  I preserved from previous version.  I'll ask Amber.